scripts/IUCNRedlistIndex/Pop_trends.R

```diff
@@ -0,0 +1,66 @@
+library(magrittr)
+library(dplyr)
+
+input <- biab_inputs()
+
+splist <- read.csv(input$splist)
+
+token <- Sys.getenv("IUCN_TOKEN")
+if (token == "") {
+  biab_error_stop("Please specify an IUCN token in your environment file")
+}
+
+
+# Extract the red list population trends
```

Comment:
Retrieving population trends is very long, and not dependent on the species list. If the API does not allow retrieving only the results for the species of interest, it should be split to a separate script to avoid being re-downloaded each time. 

By the way I just created an issue about cache expiration, to make sure these results are re-downloaded once in a while. See https://github.com/GEO-BON/bon-in-a-box-pipeline-engine/issues/284, comments welcome.